backend/db/initialize.py

```diff
@@ -22,25 +23,22 @@
 SCHEMA = CONFIG['schema']
 
 
-def create_db(con: Connection):
+def create_database(con: Connection, schema: str):
     """Create the database"""
-    if CONFIG["server"] == 'postgresql':  # postgres doesn't have create database if not exists
```

Comment:
Removed this branching. I think we're pretty much glued to Postgres, which I'm fine with.